gui/static.config.js

```diff
@@ -18,6 +18,7 @@ export default {
         component: 'src/containers/Jobs',
         getData: () => buildInfo
       },
+      {path: '/jobs/page/_jobPage_'},
```

Comment:
@rupurt am I not doing it here?

Reply:
Right you are. My bad :)

The parameter in `Layout.js` should probably change from `:page` -> `:jobPage`